resources/views/components/title/index.blade.php

```diff
@@ -1,7 +1,7 @@
 @props(['tag' => 'p'])
 <component
     is="{{ $tag }}"
-    {{ $attributes->class('text-2xl font-medium') }}
+    {{ $attributes->class('text-24 font-medium') }}
```

Comment:
text-24 isnt a default tailwind value. Can we keep using the 2xl and lg below. Because this is a public package, i'm trying to minimize the tailwind config file by keeping as close as possible to the default.